config/dev/settings.example.json

```diff
@@ -0,0 +1,4 @@
+{
+  "url": "https://example.com/path/to/the/config/endpoint",
```

Comment:
This needs to be the correct url for `dev` which we should already know.

Reply:
There also needs to be a `settings.json` for each of the other environments, because we already should know their URL.
Rename this to `settings.json`,